src/main/java/org/apache/datasketches/kll/KllDirectDoublesSketch.java

```diff
@@ -36,6 +36,8 @@
 import static org.apache.datasketches.kll.KllPreambleUtil.insertNumLevels;
 import static org.apache.datasketches.kll.KllPreambleUtil.insertPreInts;
 import static org.apache.datasketches.kll.KllPreambleUtil.insertSerVer;
+import static org.apache.datasketches.kll.KllSketch.ERRNO.ERR32;
```

Comment:
I am not sure such symbolic names are much better than magic numbers. perhaps a brief meaningful name like ERR_NOT_DIRECT or ERR_INCOMPATIBLE or something?

Reply:
OK, OK, I fixed these.
Done.

Reply:
Well, now ERRNO doesn't seem to describe it right